src/matter/cluster/Cluster.ts

```diff
@@ -98,7 +98,7 @@ export interface Cluster<F extends BitSchema, A extends Attributes, C extends Co
     id: number,
     name: string,
     revision: number,
-    features: BitSchema,
+    features: F,
```

Comment:
@mfucci I assume it should be that way, right?

Reply:
Correct!